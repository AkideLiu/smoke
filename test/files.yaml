tests:
  - name: files
    args:
      - fixtures/files
    exit-status: 0
    stdout:
      file: io/files.out
