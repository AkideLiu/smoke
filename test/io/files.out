copy a file
  succeeded
uppercase a file
  succeeded
lowercase a file
  args:   tr '[:lower:]' '[:upper:]' < input.file > output.file
  files:
    output.file:
          @@ -1 +1 @@
          -here is some text.
          +HERE IS SOME TEXT.
multiple files
  args:   echo three > one; echo two > two; echo one > three
  files:
    one:
          @@ -1 +1 @@
          -one
          +three
    three:
          @@ -1 +1 @@
          -three
          +one
multiple possible file contents
  args:   echo 0 > output.file
  files:
    output.file:
          @@ -1 +1 @@
          -1
          +0
      or: @@ -1 +1 @@
          -2
          +0
      or: @@ -1 +1 @@
          -3
          +0
increment a number
  succeeded

6 tests, 3 failures
